utils/converters.py

```diff
@@ -0,0 +1,26 @@
+from datetime import timedelta
+from re import search
+
+import disnake
+from disnake.ext.commands import Converter, converter_method
+
+from utils.errors import TimeConversionFailure
+
+
+class TimeConverter(Converter, timedelta):
```

Comment:
use a converter like durations-nlp or timestr specialised for the job

Reply:
@SpookyO work on this please :)